build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -238,8 +238,11 @@ service ActionCache {
 //   the uploaded data once uncompressed, and MUST return an
 //   `INVALID_ARGUMENT` error in the case of mismatch.
 //
-// Note that when writing compressed blobs, the `WriteRequest.write_offset`
-// refers to the offset in the uncompressed form of the blob.
+// Note that when writing compressed blobs, the `WriteRequest.write_offset` in
+// the initial request refers to the offset in the uncompressed form of the
```

Comment:
`initial request refers ...` -> `initial request in a stream refers ...` ?
I think it'd make the distinction of "request" and "stream" more clear, for the sake of resumable uploads.

Reply:
Done